src/components/DocumentationTopic.vue

```diff
@@ -29,8 +29,11 @@
         <Title :eyebrow="roleHeading">
           <WordBreak>{{ title }}</WordBreak>
           <template v-if="isSymbolBeta || isSymbolDeprecated">&nbsp;</template>
```

Comment:
We can add the space with CSS instead of a template because now VO reads "space". Somehow it was not doing it before but now it does.

<img width="632" alt="Screenshot 2022-05-02 at 20 10 37" src="https://user-images.githubusercontent.com/8567677/166303046-d8282fa6-c510-4358-a89c-f3a6a14ebb84.png">
.

Reply:
I updated the PR to use a `padding-left` to achieve this now.